homeassistant/components/assist_pipeline/pipeline.py

```diff
@@ -977,6 +977,9 @@ async def prepare_text_to_speech(self) -> None:
 
         if self.tts_audio_output is not None:
             tts_options[tts.ATTR_AUDIO_OUTPUT] = self.tts_audio_output
+        else:
+            # MP3 is preferred
+            tts_options[tts.ATTR_AUDIO_OUTPUT] = "mp3"
 
         try:
             options_supported = await tts.async_support_options(
```

Comment:
this function needs to be updated to no longer check `ATTR_AUDIO_OUTPUT`. We might even want to make sure it's gone from all TTS providers default options ?

Reply:
oh I guess we always add it to supported options